assets/src/blocks/ControlShift/firebase.js

```diff
@@ -16,4 +16,12 @@ const devConfig = {
 firebase.initializeApp(devConfig);
 firebase.analytics();
 
+export const db = firebase.firestore();
+if (location.host === 'www.planet4.test') {
```

Comment:
These host and port values should probably come from ENV vars or somewhere else.